pkg/renovate/copyright/copyright.go

```diff
@@ -29,20 +29,30 @@ import (
 // renovator.
 type CopyrightConfig struct {
 	Licenses []license.License
+	Diffs    []license.LicenseDiff
 }
 
 // Option sets a config option on a CopyrightConfig.
 type Option func(cfg *CopyrightConfig) error
 
-// WithTargetVersion sets the desired target version for the
-// bump renovator.
+// WithTargetVersion sets the licenses to be used for the
+// renovator.
 func WithLicenses(licenses []license.License) Option {
 	return func(cfg *CopyrightConfig) error {
 		cfg.Licenses = licenses
 		return nil
 	}
 }
 
+// WithLicenses sets the differences to consider for the
+// renovator.
```

Comment:
similar to above, should this comment be about what WithDiffs() does vs what WithLicenses() does?